source/grid/grid_tools.cc

```diff
@@ -3022,9 +3033,27 @@ namespace GridTools
                                 patch_to_global_tria_map_tmp.insert (std::make_pair(cell ->child(c),
                                                                                     patch_to_global_tria_map_tmp[cell]->child(c)));
 
-                                Assert(cell->child(c)->center().distance( patch_to_global_tria_map_tmp[cell]->child(c)->center())
-                                       <=1e-15*cell->child(c)->diameter(),
-                                       ExcInternalError());
+                                // We have removed the Assertion here that
```

Comment:
Just remove this whole block. Comments that explain the *history* of code are not very helpful because anyone who reads the code only sees the current state and generally doesn't care very much how this used to look in the past. If you do want to talk about something, say it in the spirit of "One could think of adding an assertion here that checks X. But this doesn't work because of Y." In other words, it explains the *current* state.